.buildkite/scripts/steps/functional/osquery_cypress.sh

```diff
@@ -4,8 +4,9 @@ set -euo pipefail
 
 source .buildkite/scripts/common/util.sh
 
+export BUILD_TS_REFS_DISABLE=false
 .buildkite/scripts/bootstrap.sh
-.buildkite/scripts/download_build_artifacts.sh
```

Comment:
Why the changes in this file? For functional tests, we prefer to run against the distribution, not the development environment.

Reply:
This question was also asked on another PR https://github.com/elastic/kibana/pull/122023#discussion_r775554087.

Reply:
@tylersmalley where we are trying to adopt https://www.npmjs.com/package/cypress-react-selector but it requires the react components to keep `displayName`s, otherwise, Cypress will not be able to locate them properly. To do it the right way I believe we should modify webpack/babel config to don't remove those values from the build, but like in a code coverage case, first, we would like to check if that approach will work and scale before we start changing the core pipelines

Reply:
Thanks for the details, that makes sense. Are there any discussions on the trade-offs to using that package vs not testing the build? I don't think we would want to modify Babel to include those in the production bundles. As it would increase bundle sense for something we are only using under test.

Reply:
We did not have a discussion about the trade-offs yet, mostly because we didn't know what impact it will have on the pipelines and if it's going to improve the dev experience as we think it will.
Also, this pipeline is going to run rather rarely (runs only when our plugin was changed
https://github.com/elastic/kibana/blob/main/.buildkite/scripts/pipelines/pull_request/pipeline.js#L95-L100
) as we are a small, 2 UI people team, so we don't expect too many PR open in the near future, so we thought that the impact of that experiment will be rather small in the context of the CI.

Reply:
Ok, thanks.